brave-instrumentation/aws-java-sdk-core/src/test/java/brave/instrumentation/aws/AwsClientTracingTest.java

```diff
@@ -94,6 +102,16 @@ public void testSpanCreatedAndTagsApplied() throws InterruptedException {
     assertThat(sdkSpan.name()).isEqualToIgnoringCase("aws-sdk");
   }
 
+  @Test
+  public void buildingAsyncClientWithEmptyConfigDoesNotThrowExceptions() {
+    Tracing tracing = tracingBuilder().build();
+    HttpTracing httpTracing = HttpTracing.create(tracing);
+    environmentVariables.set("AWS_REGION", "us-east-1");
+
+    AmazonDynamoDBAsync asyncClient =
+        AwsClientTracing.create(httpTracing).build(AmazonDynamoDBAsyncClientBuilder.standard());
+  }
```

Comment:
@sdavids13 can you please validate this test is what you meant in the issue you filed

Reply:
👍